dev/NavigationView/NavigationView.xaml

```diff
@@ -538,10 +558,10 @@
                                             Grid.Row="4" />
 
                                         <!-- "Non header" content -->
-                                        <Grid x:Name="ItemsContainerGrid" Grid.Row="6" Margin="0,0,0,8">
+                                        <Grid x:Name="ItemsContainerGrid" Grid.Row="6">
                                             <Grid.RowDefinitions>
-                                                <RowDefinition Height="Auto"/> <!-- MenuItems -->
-                                                <RowDefinition Height="*"/> <!-- Fill spacing -->
+                                                <RowDefinition Height="*"/> <!-- MenuItems -->
```

Comment:
Please, correct me if I'm wrong, but in .cpp in UpdatePaneLayout():
![image](https://user-images.githubusercontent.com/21356912/114441163-e2040c00-9b7f-11eb-87c1-06e4f257fdb9.png)
`menuItemsActualHeight` won't return the correct value if you set Height="*" because it will take all the space that it can and ActualHeight() will return that value. The logic in UpdatePaneLayout won't be working correctly then :)

Reply:
It actually needs the actual height value where the menuItems row take up the remainder of the space. Otherwise, the spacing go towards FillSpacing, and that throws off the entire design grid.